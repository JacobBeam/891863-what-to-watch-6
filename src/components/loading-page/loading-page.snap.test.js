import React from 'react';
import {render} from '@testing-library/react';
import LoadingPage from './loading-page';


it(`Should Mistakes render correctly`, () => {
  const {container} = render(<LoadingPage/>);
  expect(container).toMatchSnapshot();
});
