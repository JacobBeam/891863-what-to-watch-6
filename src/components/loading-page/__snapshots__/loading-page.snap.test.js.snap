// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Mistakes render correctly 1`] = `
<div>
  <div
    class="user-page"
  >
    <header
      class="page-header user-page__head"
    >
      <div
        class="logo"
      >
        <a
          class="logo__link"
        >
          <span
            class="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            class="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            class="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <h1
        class="page-title user-page__title"
      />
    </header>
    <div
      class="sign-in user-page__content"
    >
      <h2>
        Wait, please. Loading...
      </h2>
    </div>
    <footer
      class="page-footer"
    >
      <div
        class="logo"
      >
        <a
          class="logo__link logo__link--light"
        >
          <span
            class="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            class="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            class="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        class="copyright"
      >
        <p>
          Â© 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>
</div>
`;
